tests/fe/bdm_16.cc

```diff
@@ -0,0 +1,63 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2003 - 2017 by the deal.II authors
```

Comment:
since it's `2017` today and you contribute this file today, i would say change `2003 - 2017` to `2017` :smile: